app/index/template.hbs

```diff
@@ -35,7 +35,7 @@
 		{{/if}}
 
 		<p>
-			Looking for machine-readable JSON instead? The {{json-endpoint "/how-it-works/#slide-3" "Datastore API"}} can provide all these {{json-endpoint "operators"}} or all these {{json-endpoint "feeds"}}.
+			Looking for machine-readable JSON instead? The {{json-endpoint "/how-it-works/#slide-3" "" "Datastore API"}} can provide all these {{json-endpoint "operators" "" "operators"}} or all these {{json-endpoint "feeds" "" "feeds"}}.
```

Comment:
Since the link to `/how-it-works/` isn't actually going to the API, I recommend replacing that with plain old HTML: `<a href="/how-it-works/#slide-3">...`